src/main/scala/com/databricks/spark/corenlp/functions.scala

```diff
@@ -15,9 +15,10 @@ import edu.stanford.nlp.util.Quadruple
 import org.apache.spark.sql.functions.udf
 
 /**
- * A collection of Spark SQL UDFs that wrap CoreNLP annotators and simple functions.
- * @see [[edu.stanford.nlp.simple]]
- */
+  * A collection of Spark SQL UDFs that wrap CoreNLP annotators and simple functions.
```

Comment:
nit: Please keep the code style consistent. We follow Spark code style http://spark.apache.org/contributing.html. I understand it is a little annoying without an auto formatter. But your time would be greatly appreciated:)

Reply:
No problem, hope I didn't make other troubles.